helm/prometheus-operator/values.yaml

```diff
@@ -37,9 +37,8 @@ resources: {}
   #   cpu: 100m
   #   memory: 50Mi
 
-## The prometheus operator will create an endpoint for the kubelet.
-## Format: namespace/name
+## If enabled, prometheus-operator will create a service for scraping kubelets
 ##
-kubeletObject:
+kubeletService:
   enable: true
-  object: kube-system/kubelet
+  name: kube-system/kubelet
```

Comment:
since we have this abstraction layer here it would make sense to divide this into two fields: name and namespace. Then we can do the concatenation in the actual template.